src/05-derivatives/01-introduction.md

```diff
@@ -97,24 +122,13 @@ derivative includes REQUIRED metadata fields in which case a JSON file is also
-
-### CoordinateSystem key allowed values
-
-In addition to values defined in
-[Appendix VII Table "Template based Coordinate Systems"](../99-appendices/08-coordinate-systems.md).
-
-| **Value name** | **Description**                                                                                                             |
-| -------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| Unknown        | The coordinate system is unknown.                                                                                           |
-| Custom         | A custom coordinate system that is not in alignment (dimensions, axis orientation, unit) with any device coordinate system. |
+| **Key name**   | **Description**                                                                                                                                                                                                                                                                                                                                       |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Description    | RECOMMENDED. Free-form natural language description of the nature of the file.                                                                                                                                                                                                                                                                        |
+| Sources        | OPTIONAL. A list of paths relative to dataset root pointing to the file(s) that were directly used in the creation of this derivative. For example in a chain of A->B->C, “C” should only list “B” as Sources, and “B” should only list “A” as Sources. However in case X and Y jointly contribute to Z, then “Z” should list “X” and “Y” as Sources. |
+| RawSources     | OPTIONAL. A list of paths relative to dataset root pointing to the BIDS-Raw file(s) that were used in the creation of this derivative.                                                                                                                                                                                                                |
+| ReferenceMap   | REQUIRED when a custom template or file is used. A path to a file that was used as, or can be used as, a reference image for determining the coordinate space of this file. The path should start with a `/` and should be relative to the root of the dataset.                                                                                       |
```

Comment:
I feel pretty strong about not going forward with ``ReferenceMap`` (see https://github.com/chrisfilo/bids-specification/pull/13).

Reply:
I hear you and I remember, just wanted to address one thing at a time. Renaming `ReferenceMap` to something else does not influence other topics discussed in this PR. We'll get back to that very soon. Promise!

Reply:
Oh, apologies for pushing back. Still getting used to this style of working.

Reply:
No worries! We all are! I appreciate your feedback no matter which channel it comes from.